src/components/GraphFilter/LayoutDropdown.tsx

```diff
@@ -20,33 +21,49 @@ export class LayoutDropdown extends React.Component<LayoutDropdownProps, LayoutD
     ['dagre', 'Dagre'],
     ['klay', 'Klay']
   ];
+  static InitialLabel = 'Select Graph Type';
+
   constructor(props: LayoutDropdownProps) {
     super(props);
     this.state = {
-      currentGraphType: props.initialLayout
+      currentGraphType: props.initialLayout,
+      currentGraphTypeLabel: this.getGraphTypeName(props.initialLayout)
     };
   }
 
+  getGraphTypeName = (graphType: string) => {
+    let label = LayoutDropdown.InitialLabel;
+    LayoutDropdown.GraphTypes.map(subarray => {
+      if (subarray[0] === graphType) {
+        label = subarray[1];
+      }
+    });
+    return label;
+  };
+
   layout = (graphType: string) => {
-    this.setState({ currentGraphType: graphType });
+    this.setState({ currentGraphType: graphType, currentGraphTypeLabel: this.getGraphTypeName(graphType) });
```

Comment:
Does `currentGraphTypeLabel` need to be stored in the state since it is just derived from `graphType`?

Reply:
Mmm No ^^ We can call him in the element. You are right